src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const employeeList = document.querySelector('ul');
+
+function sortList(listContainer) {
+  const listElements = Array.from(listContainer.children);
+
+  listElements.sort((a, b) => {
+    const salaryA = parseSalary(a.dataset.salary);
+    const salaryB = parseSalary(b.dataset.salary);
```

Comment:
Ensure that the HTML elements have the `data-salary` attribute set correctly. If this attribute is missing or incorrectly formatted, the `parseSalary` function may not work as expected.